File: video/v1/subscribed_track/update-default/update-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

subscribed_track = @client.video.rooms('RMaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')
                          .participants('PAaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')
                          .subscribed_tracks
                          .update(track: 'MTaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')

puts subscribed_track.sid

File: preview/deployedDevices/key/update-default/update-default.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Preview.DeployedDevices.Fleet;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var key = KeyResource.Update(
            pathFleetSid: "FLaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
            pathSid: "KYaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
        );

        Console.WriteLine(key.Sid);
    }
}
